storage/database.go

```diff
@@ -304,7 +348,7 @@ func (d *db) splayedTick() {
 	}
 }
 
-func (d *db) nextUniqueIndex() uint64 {
+func (d *db) next() uint64 {
```

Comment:
Any reason to change this?  `next()` gives no context for what it's returning or who might use this function.